gp-templates/translations.php

```diff
@@ -165,9 +174,20 @@
 		 *
 		 * This action is inside a DL element.
 		 *
+		 * @deprecated 2.1.0 Call gp_translation_set_sort_form instead
```

Comment:
@ocean90 that's a nice and clean workaround, but I agree with @toolstack that it's better being explicit here.